### PR TITLE
Added functionality to increment weboploy wins

### LINKED ISSUE #72: Add webopoly win incrementing functionality to timbot
Need to add a system that allows users, with a consensus, to increment a user's Webopoly win count using the db function introduced in #71

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #72

### CODE REVIEW

File: src/database.py [hunk 1]
```diff
@@ -7,11 +7,8 @@ def get_webopoly_standings(conn):
...
-        conn.commit()
-        cursor.close()
-        return 1
-    except Exception as e:
-        return e
+    cursor = conn.cursor()
+    cursor.execute('UPDATE webopoly SET wins = wins + 1 WHERE webopoly.uid = (SELECT uid FROM users WHERE name = "{}")'.format(name))
+    conn.commit()
+    cursor.close()
+    return 1
```
Comment (timflannagan) [inline src/database.py]: Why are you returning anything here?

Comment (nathan-weinberg) [reply src/database.py]: Idk I wrote that a while ago

Comment (timflannagan) [reply src/database.py]: lol

File: src/database.py [hunk 2]
```diff
@@ -7,11 +7,8 @@ def get_webopoly_standings(conn):
...
-        conn.commit()
-        cursor.close()
-        return 1
-    except Exception as e:
-        return e
+    cursor = conn.cursor()
+    cursor.execute('UPDATE webopoly SET wins = wins + 1 WHERE webopoly.uid = (SELECT uid FROM users WHERE name = "{}")'.format(name))
+    conn.commit()
+    cursor.close()
+    return None
```
Comment (Daniel-Pivonka) [inline src/database.py:14]: no need for a return

Comment (timflannagan) [reply src/database.py:14]: Daniel that's good practice.

Comment (nathan-weinberg) [reply src/database.py:14]: smh Dan doesn't know Python conventions

**Review by timflannagan — APPROVED** [review]
superb work as always

**Pull request merged**